airflow-core/src/airflow/serialization/serialized_objects.py

```diff
@@ -1660,13 +1730,34 @@ def detect_dependencies(cls, op: SdkOperator) -> set[DagDependency]:
 
     @classmethod
     def _is_excluded(cls, var: Any, attrname: str, op: DAGNode):
+        """
+        Determine if a variable is excluded from the serialized object.
+
+        :param var: The value to check. [var == getattr(op, attrname)]
+        :param attrname: The name of the attribute to check.
+        :param op: The operator to check.
+        :return: True if a variable is excluded, False otherwise.
+        """
+        schema_defaults = cls.get_operator_defaults_from_schema()
+
+        if attrname in schema_defaults:
```

Comment:
Worth checking type. Example if default is empty and `False` is passed.

Reply:
`if not var:` can not be used since it would also exclude False, 0 or empty string